apps/BraynsViewer/BraynsViewer.cpp

```diff
@@ -101,6 +97,10 @@ void BraynsViewer::keypress(char key, const Vector2f& where)
         BRAYNS_INFO << "Default renderer activated" << std::endl;
         renderingParams.setRenderer("exobj");
         break;
+    case '7':
```

Comment:
Agreed! I think the nice way to do it is to create a 'key manager' than can be shared between the UI and Deflect so that we get the same interaction on all displays. TODO then ;-)